content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/csi-node-expand-secret.md

```diff
@@ -0,0 +1,13 @@
+---
+title: CSINodeExpandSecret
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+<!--
+Enable passing secret authentication data to a CSI driver for use
+ during a `NodeExpandVolume` CSI operation.
+-->
+允许在 `NodeExpandVolume` CSI 操作期间将 Secret
+身份验证数据传递到 CSI 驱动以供后者使用。
```

Comment:
Refer the [blog](https://kubernetes.io/blog/2022/09/21/kubernetes-1-25-use-secrets-while-expanding-csi-volumes-on-node-alpha/)

```suggestion
允许在 `NodeExpandVolume` CSI 操作期间，
将保密的身份验证数据传递到 CSI 驱动以供后者使用。
```